object_visualizer/include/object_visualizer/object_visualizer_config.hpp

```diff
@@ -33,6 +33,9 @@ namespace object_visualizer
     //! If true then RViz markers will be forwarded for received carma_perception_msgs/RoadwayObstacleList messages
     bool enable_roadway_objects_viz = true;
 
+    // Unsigned int: Visualization marker shape: CUBE=1, SPHERE=2, CYLINDER=3
+    uint8_t marker_shape = 2;
```

Comment:
We need to add this member variable's value to the stream operator overload.

Reply:
added